it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SignOutControllerISpec.scala

```diff
@@ -23,15 +24,15 @@ class SignOutControllerISpec extends BaseISpec {
 
   "redirect to SOS" should {
     "redirect to SOS page" in {
-      val result = await(controller.redirectToSos(authenticatedRequest()))
+      val result = await(controller.redirectToSos(authenticatedRequest(subscribingAgentEnrolledAsHMRCASAGENT)))
```

Comment:
It's not really relevant to most of the tests in this class which just need a simple logged in user, but using `subscribingAgentEnrolledForNonMTD` rather than `subscribingAgentEnrolledAsHMRCASAGENT` in the tests might be more in line with the journeys that the tests are targetting.